examples/browser/package.json

```diff
@@ -7,6 +7,8 @@
     "frontend": {
       "config": {
         "applicationName": "Theia Browser Example",
+        "defaultTheme": "dark",
```

Comment:
The PR description states:

> make sure that color and icon themes are having default values.

You have defined a default, what is the expected behavior?

Can you update the electron example as well? Thanks!

Reply:
> You have defined a default, what is the expected behavior?

Let me try to clarify if it was not already clear; does it also work if we remove the default theme and icon config from the example's `package.json`?

Reply:
they are optional, i will remove them from here

Reply:
removed